config/deprecation-workflow.js

```diff
@@ -1,6 +1,7 @@
 window.deprecationWorkflow = window.deprecationWorkflow || {};
 window.deprecationWorkflow.config = {
   workflow: [
-    { handler: 'silence', matchId: 'ember-routing.route-router' }
+    { handler: 'silence', matchId: 'ember-routing.route-router' },
+    { handler: 'silence', matchId: 'ember-views.curly-components.jquery-element'} //Depreciating the jquery warnings until we have moved away from jquery or used native jquery in the app.
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [A space is required before '}'.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=4886973)